Chap_API_Job_Mgmt.tex

```diff
@@ -1167,3 +1167,281 @@ \subsection{Log attributes}
 }
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Resource Usage}
+\label{api:resusage}
+
+Applications and tools often want to obtain a snapshot of the resources utilized by individual
+processes and/or nodes in their job. \ac{PMIx} provides attributes for querying such information
+and accompanying structures for the results.
+
+\subsection{Process statistics}
+\label{api:pstats:struct}
+
+The process statistics structure contains information available from many \acp{OS}. The field
+corresponding to any information that is not available will be set to zero.
+
+\declarestruct{pmix_proc_stats_t}
+
+\versionMarker{4.1}
+\cspecificstart
+\begin{codepar}
+typedef struct pmix_proc_stats \{
+    /* process ident info */
+    char *node;
+    pmix_proc_t proc;
+    pid_t pid;
+    char *cmd;
+    /* process stats */
```

Comment:
Do we have a link to more descriptions on each of these stats? We take them from Linux so a pointer to a good manpage description would be helpful to someone new to these fields.